lua/nvim_lsp/configs.lua

```diff
@@ -33,26 +33,26 @@ function configs.__newindex(t, config_name, config_def)
   -- The config here is the one which will be instantiated for the new server,
   -- which is why this is a function, so that it can refer to the settings
   -- object on the server.
-  local function add_callbacks(config)
-    config.callbacks["window/logMessage"] = function(err, method, params, client_id)
+  local function add_handlers(config)
+    assert(not lsp.callbacks, "lsp.callbacks has been deprecated. See here for more: https://github.com/neovim/neovim/pull/12655")
```

Comment:
Oh, we should also chdck that config.callbacks is nil.

Sorry to be so pedantic about this.

Reply:
It's no problem! We want to make sure it's right before it goes someplace people use it everyday.

I'll add it in a little bit and push, just leaving this here to say I've seen it.

Reply:
Cool, ping me on Monday if I haven't looked again. Idk what my schedule is tomorrow.